config-linux.md

```diff
@@ -384,6 +384,31 @@ The following parameters can be specified to set up the controller:
     }
 ```
 
+## <a name="configLinuxVTPMs" />vTPMs
+
+**`vtpms`** (array of objects, OPTIONAL) lists a number of emulated TPMs that
+will be made available to the container.
+
+Each entry has the following structure:
+
+* **`statePath`** *(string, REQUIRED)* - full path to a directory where the vTPM is to write its persistent state into
```

Comment:
From opencontainers/runc#1591, it looks like you're writing a `swtpm` [PID][1] and [log][2] files to this directory.  But this spec PR doesn't explain that, so folks consuming a generic runtime (which may or may not be runc) won't be able to rely on that.  Do you have a use-case for the runtime-caller reaching into this directory to check those files?  If so, I think you should specify the files here.  If not, I think we can drop this property from runtime-spec, and generate it on the fly (or based on runtime-specific options, like runc's existing `--root` option) instead of having it be configured here.

[1]: https://github.com/opencontainers/runc/pull/1591/files#diff-3076cfa104fbee0471fea515edf4f255R154
[2]: https://github.com/opencontainers/runc/pull/1591/files#diff-3076cfa104fbee0471fea515edf4f255R158